dist/mk-docker-opts.sh

```diff
@@ -101,3 +101,15 @@ done
 if [ "$combined_opts" = true ]; then
 	echo "${combined_opts_key}=\"${docker_opts}\"" >>$docker_env
 fi
+
+if [ "x${FLANNEL_SUBNET}" != "x" ];then
+    dot_four=`echo ${FLANNEL_SUBNET}|cut -d. -f4|cut -d/ -f1`
+    dot_four=$((${dot_four}-1))
+    subnets_pre=`echo ${subnets}|cut -d. -f 1,2,3`
```

Comment:
subnets does not exist, or?

Reply:
> subnets does not exist, or?

This is a way for me to empty the string. Please ignore this bad expression.